reana_commons/config.py

```diff
@@ -362,3 +362,9 @@ def load(cls, kind):
 }
 
 """HTCondor job flavours and their respective runtime in seconds."""
+
+REANA_KUBERNETES_JOBS_KEEP_ON_STATUS_LIST = os.getenv(
```

Comment:
(1) We already have `REANA_RUNTIME_KUBERNETES_NAMESPACE`, so what about using the same `REANA_RUNTIME...` starting prefix, and renaming the variable to something perhaps more readable, like `REANA_RUNTIME_KUBERNETES_KEEP_ALIVE_JOBS_WITH_STATUSES`  which might be perhaps more understandable?  (...taking into account that plural "statuses" somehow indicates a list of status values...)

(2) The help string could be also amended to be perhaps more readable, as follows:  

Keep alive Kubernetes user runtime jobs, both batch workflow jobs and invididual step jobs, after their termination, provided that their statuses are matching some of the comma-separated values specified here. Example: "finished,failed". 

It still reads perhaps a bit too convoluted...

(3) What about the default value? I guess we remain auto-killing them, as before... so perhaps let's just say in the docstring at the end that "By default, the jobs are removed after FOO_TTL seconds."

Reply:
Thanks for the review. I've updated the PR set and rechecked that everything works. Regarding the docstring I've slightly modified your suggestion, please let me know what you think @tiborsimko.